node/core/provisioner/Cargo.toml

```diff
@@ -16,4 +16,7 @@ polkadot-node-subsystem-util = { path = "../../subsystem-util" }
 [dev-dependencies]
 lazy_static = "1.4"
 sp-core = { git = "https://github.com/paritytech/substrate", branch = "master" }
-tokio = "0.2"
+sp-application-crypto = { git = "https://github.com/paritytech/substrate", branch = "master" }
+sc-keystore = { git = "https://github.com/paritytech/substrate", branch = "master" }
+tokio = { version = "0.2.22", features = ["time"] }
```

Comment:
Why is the `time` feature` needed?

Reply:
There are some tests that use tokio::time which has been moved in recent versions to the "time" feature.

Reply:
Opened #1745 to simplify this.

Reply:
now that #1745, we can remove tokio here